pkg/graph/schema.resolvers_helpers.go

```diff
@@ -5,7 +5,7 @@ import (
 	"github.com/cmsgov/mint-app/pkg/models"
 )
 
-// ConvertToModelPlan takes an auto-generated model plan input and converts it to a hand-written one
```

Comment:
I don't think that these comments, variable name changes / or anything not directly concerning a `Document` object should be touched in this branch. It increases the chance of conflicts, and is not the work scoped for this branch. If the change should be made, I think we should make a separate branch, or at least a ticket to do it in the future.